.github/workflows/ci_emulator_run.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     timeout-minutes: 45
     env:
       VAB_FLAGS: -cg -f '-d no_load_styles' -v 3 --api 30 --build-tools 29.0.0 --device auto --log-clear --archs x86_64
+      V_FLAGS: -no-parallel
```

Comment:
```suggestion
      VFLAGS: -no-parallel
```

Reply:
Oh dear - such a subtle mistake. Fixed in [ed96dd2](https://github.com/vlang/vab/pull/284/commits/ed96dd256ffdc992663ef3179abc3ab76c27a815)